sdk/inc/azure/core/az_span.h

```diff
@@ -485,7 +514,7 @@ typedef struct
  * This method must never return an empty #az_span, unless the requested buffer size is not
  * available. In which case, it must return an error #az_result.
  *
- * @remarks The caller must check the return value using #az_succeeded() before continuing to use
+ * @remarks The caller must check the return value using #az_succeeded before continuing to use
```

Comment:
Revert, `az_succeeded` is a method.
```suggestion
 * @remarks The caller must check the return value using #az_succeeded() before continuing to use
```